src/geojson-layer.ts

```diff
@@ -137,6 +168,20 @@ export default class GeoJSONLayer extends React.Component<Props, {}> {
     map.removeSource(this.id);
     map.off('styledata', this.onStyleDataChange);
     this.layerIds.forEach(lId => map.removeLayer(lId));
+
+    for (const type in typeToLayerLUT) {
```

Comment:
Could you use `Object.keys(typeToLayerLUT).forEach ` to keep the coding style consistent with the rest of the codebase? Same for eventToHandler and same for lines 223 to 227